resources/deployments/controller/controller_deployments.go

```diff
@@ -173,8 +173,12 @@ func (d *DeploymentsController) AbortDeployment(w rest.ResponseWriter, r *rest.R
 	d.view.RenderEmptySuccessResponse(w)
 }
 
+const (
+	GetDeploymentForDeviceQueryArtifact   = "artifact_name"
+	GetDeploymentForDeviceQueryDeviceType = "device_type"
```

Comment:
What do you mean by use? Extract it from url.Values just for the sake of getting it? Because there's no other use for it at this time.